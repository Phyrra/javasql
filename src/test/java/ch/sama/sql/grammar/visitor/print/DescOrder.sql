select *
from t
order by f desc