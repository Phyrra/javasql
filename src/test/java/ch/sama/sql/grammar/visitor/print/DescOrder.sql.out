statement
	select
		value
			all
	from
		source
			primary
				t
	order
		order
			desc
				primary
					field
						f